vnext/JSI/Desktop/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="ChakraCore.Debugger" version="0.0.0.39" targetFramework="native" />
```

Comment:
Is this truly a dependency for JSI.Desktop?

Reply:
Yes. ChakraJsiRuntime uses it.

---
In reply to: [330305323](https://github.com/microsoft/react-native-windows/pull/3296#discussion_r330305323) [](ancestors = 330305323)